ra2ce/graph/hazard/hazard_files.py

```diff
@@ -0,0 +1,11 @@
+from pathlib import Path
+from typing import Optional
+
+from attr import dataclass
+
+
+@dataclass
+class HazardFiles:
+    tif: list[Optional[Path]] = None
+    shp: list[Optional[Path]] = None
+    table: list[Optional[Path]] = None
```

Comment:
It does not make much sense to me to have an item in a list which is optional, either you have it or not. Also you are giving as a default `None`. Simply by doing `list[Path] = field(default_factory=list)` would be sufficient